SDK/AppCenter/Microsoft.AppCenter.UWP/Utils/DeviceInformationHelper.cs

```diff
@@ -14,8 +14,6 @@ namespace Microsoft.AppCenter.Utils
     /// </summary>
     public class DeviceInformationHelper : AbstractDeviceInformationHelper
     {
-        public static event EventHandler InformationInvalidated;
```

Comment:
Removing this line makes UWP no longer compile.

There are also 3 relevant/affected UWP tests for this change
 * "Microsoft.AppCenter.Test.UWP.AppCenterTest" `SetCountryCode` 
 * "Microsoft.AppCenter.Test.UWP.Utils.DeviceInformationHelperTest" `VerifyCarrierCountry`
 * "Microsoft.AppCenter.Test.UWP.Utils.DeviceInformationHelperTest" `VerifyScreenSizeProvider`

The first 2 tests above are probably duplicates of the new test.

Reply:
Ah. That comes after the merge with new changes. Thanks

Reply:
Hm. I have no idea how `VerifyScreenSizeProvider` worked before. It heavily relied on tests execution order.

Reply:
cc @achocron

Reply:
Aha! It's already fixed - #988

Reply:
I'm keeping my variant of a fix if anyone doesn't mind